repose-aggregator/src/docs/asciidoc/filters/cors.adoc

```diff
@@ -26,18 +33,45 @@ All of these headers will be supplied by the client making the request.
 | None required
 a| * `Origin`
    * `Access-Control-Request-Method`
-a| * `Origin`
+a| * `Origin`*
 |===
 
-=== Created
-The CORS filter wil not add any headers to the response for non-CORS requests.
-The filter will add the following headers to the response for these scenarios when the request is allowed to proceed:
+*{sp}If the `Origin` header matches the original `Host` header in the request, it will be considered a non-CORS request.
```

Comment:
This is kind of odd. I get that the {sp} attribute is so that this isn't made into a list, but...

Would a footnote work here? I mean, it would work, but is it displayed in a way that you find acceptable?
http://asciidoctor.org/docs/user-manual/#user-footnotes